Sources/StreamChatUI/StreamSwiftyMarkdown/SwiftyMarkdown/SwiftyLineProcessor.swift

```diff
@@ -244,6 +249,15 @@ class SwiftyLineProcessor {
         return foundAttributes
     }
     
+    func processOrderedListRegex(_ text: String) -> String {
```

Comment:
can we also add a snapshot test with the example you have in the PR description?

Reply:
added 👍